.travis.yml

```diff
@@ -1,13 +1,7 @@
-language: node_js
-
-node_js:
-  - '8'
-
 services:
   - docker
-  - rabbitmq
 
-script: npm run test:ci
+script: npm run test
```

Comment:
Should be able to just use `npm test` here. Also - are we guaranteed `npm` is available if we don't specify the language/node_js version? For consistent behavior, might still be worth specifying.